addons/binding/org.openhab.binding.unifi/src/main/java/org/openhab/binding/unifi/internal/api/model/UniFiClient.java

```diff
@@ -50,6 +50,12 @@
     @JsonAdapter(UniFiTimestampDeserializer.class)
     protected Calendar lastSeen;
 
+    protected boolean blocked;
+
+    public boolean isBlocked() {
+        return this.blocked;
+    }
+
     protected UniFiDevice device;
```

Comment:
The getters / setters should not be mixed with the property declarations.

Should be something like...

```
public class UniFiClient {

    // properties

    protected boolean blocked;

    // getters / setters

    public boolean isBlocked() { 
        return blocked;
    }

    public void setBlocked(boolean blocked) {
        this.blocked = blocked;
    }

    ...

    // toString() overrides are usually found at the end of a class declaration
    public String toString() {
        ...
    }

}
```

Reply:
Changed!